### DISCUSSION TITLE
How to install kairos on incus?

### DISCUSSION THREAD

**Discussion by onlookor** [kairos-io/kairos#3011] [Q&A]
I failed at the first step: import image.
I downloaded a kairos ISO image from github release, tried the command:
`incus image import kairos-alpine-3.19-core-amd64-generic-v3.2.1.iso`
And get an error: `Unsupported compression`.
Some guide told me to use the squashfs image, but `incus image import` reported an error:
`Metadata tarball is missing metadata.yaml`
So, I have to make up a metadata.yaml, but it doesn't work anyway.

**Comment by j-landru** [comment]
In the past, I had some success to run abcdic on LXD (see polymorphic OCI OS images with Kairos #1875 context), I don't know if it's still the case with more recent version or with incus. I published some howto details (see end of that page : https://framagit.org/j-landru/abcd/-/blob/main/README.sav20240530.md?ref_type=heads#lxd- ), maybe some tips are still useful.
